packages/beacon-node/src/network/core/networkCore.ts

```diff
@@ -390,18 +390,29 @@ export class NetworkCore implements INetworkCore {
     await this.libp2p.hangUp(peerIdFromString(peerIdStr));
   }
 
+  private _dumpPeer(peerIdStr: string, connections: Connection[]): routes.lodestar.LodestarNodePeer {
+    const peerData = this.peersData.connectedPeers.get(peerIdStr);
+    return {
+      ...formatNodePeer(peerIdStr, connections),
+      agentVersion: peerData?.agentVersion ?? "NA",
+      status: peerData?.status ? ssz.phase0.Status.toJson(peerData.status) : null,
+      metadata: peerData?.metadata ? ssz.altair.Metadata.toJson(peerData.metadata) : null,
```

Comment:
once Fusaka goes live we should update this to use `fulu.Metadata`, or pass down fork info and do if-else here